Duplicate removal about invoice print and adding the "Payment actions" block

EvheniiaKolkhiieva opened the pull request:
## Purpose of this pull request

<!-- REQUIRED Describe the goal and the type of changes this pull request covers. Tell us what changes are you making and why. -->
1 Case:  Add the "Payment actions" block 

2 Case: Since "Printing Invoices" is described in a separate subcategory, it should not be described in the "Creating an Invoice" topic, this only loads the document. This PR removes unnecessary duplicate information
![image](https://user-images.githubusercontent.com/78729171/114937730-d1d18280-9e46-11eb-9db0-6ef0ef583494.png)


## Magento release version

<!-- Use this section to indicate which Magento release(s) are affected by the content changes. -->

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

<!-- REQUIRED List HTML links for affected pages on https://docs.magento.com -->

- https://docs.magento.com/user-guide/sales/invoice-create.html
- https://docs.magento.com/user-guide/sales/refunds-customer-account.html?itm_source=merchdocs&itm_medium=quick_search&itm_campaign=federated_search&itm_term=refund

<!-- OPTIONAL - REMOVE THIS SECTION IF NOT USED. What other information can you provide? -->

<!--
If you are fixing a GitHub issue, note it using GitHub keyword format (https://help.github.com/en/articles/closing-issues-using-keywords#closing-an-issue-in-a-different-repository) to close the issue when this pull request is merged. Example: `Fixes #1234`

`master` is the default branch. Merged pull requests to `master` go live on the site automatically. Any requested changes to content on the `master` branch must be related to the released codebase. Any content related to future releases goes in a release integration branch.

See Contribution guidelines (https://github.com/magento/merchdocs/blob/master/.github/CONTRIBUTING.md) and wiki (https://github.com/magento/merchdocs/wiki) for more information.
-->

dobooth commented:
Any chance you could remove that Acrobat screenshot from what appears to be 1998 from the Printing Invoices topic?

EvheniiaKolkhiieva commented:
@dobooth yes, for sure. 

Should I just delete the image in the additional commit or add an updated modern one?

EvheniiaKolkhiieva commented:
Hi @dobooth,
I have deleted the Acrobat screenshot from the Printing invoices topic.
Could you please approve?

Thank you!

leslietilling commented on src/sales/refunds-customer-account.md:
```diff
@@ -29,3 +29,16 @@ If a refund has been issued for an order, customers can view the refund informat
+
+You can configure payment actions for your specific [payment method]({% link configuration/sales/payment-methods.md %}). Each payment method has a different set of payment actions.
+
+|Payment Action |Description
```
Sentence case for column headings

```suggestion
|Payment action |Description
```

EvheniiaKolkhiieva replied:
Done ☑️

leslietilling commented on src/sales/refunds-customer-account.md:
```diff
@@ -29,3 +29,16 @@ If a refund has been issued for an order, customers can view the refund informat
+|--- |---
+|Capture Online |When the invoice is submitted, the system captures the payment from the third-party payment gateway. You have the ability to create a credit memo and void the invoice.
+|Capture Offline |When the invoice is submitted, the system does not capture the payment. It is assumed that the payment is captured directly through the gateway, and you no longer have the option to capture this payment through Magento. You have the ability to create a credit memo, but you do not have the option to void the invoice. (Even though the order used an online payment, the invoice is e ...
+|Not Capture |When the invoice is submitted, the system does not capture the payment. It is assumed that you will capture the payment through Magento at a later date. There is a Capture button in the completed invoice. Before capturing, you are able to cancel the invoice. After capturing you are able to create a credit memo and void the invoice.
```
```suggestion
|Not Capture |When the invoice is submitted, the system does not capture the payment. It is assumed that you will capture the payment through Magento at a later date. There is a _Capture_ button in the completed invoice. Before capturing, you are able to cancel the invoice. After capturing, you are able to create a credit memo and void the invoice.
```

EvheniiaKolkhiieva replied:
Done ☑️

leslietilling commented on src/sales/refunds-customer-account.md:
```diff
@@ -29,3 +29,16 @@ If a refund has been issued for an order, customers can view the refund informat
+|Not Capture |When the invoice is submitted, the system does not capture the payment. It is assumed that you will capture the payment through Magento at a later date. There is a Capture button in the completed invoice. Before capturing, you are able to cancel the invoice. After capturing you are able to create a credit memo and void the invoice.
+
+{:.bs-callout-info}
+Do not select the _Not Capture_ option unless you are certain that you are going to capture the payment through Magento at a later date. You cannot create a credit memo until the payment has been captured using the Capture button.
```
```suggestion
**_Do not_** select the _Not Capture_ option unless you are certain that you are going to capture the payment through Magento at a later date. You cannot create a credit memo until the payment has been captured using the _Capture_ button.
```

EvheniiaKolkhiieva replied:
Done ☑️

leslietilling reviewed (changes requested):
Just a few minor edits and this one looks good.

EvheniiaKolkhiieva commented:
Hi @leslietilling,

I have implemented your edits, could you please approve? 

Thank you!

Re: pay attention, seems the "Adding the "Payment actions block " commit mistakenly got into this PR, since it does not apply to the description above, but I updated the description to combine both edits

leslietilling commented:
I will fix the conflict in the team fork.

leslietilling commented:
@magento import code to magento-devdocs/merchdocs

magento-engcom-team commented:
@leslietilling the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1189`.

EvheniiaKolkhiieva commented:
Thank you a lot @leslietilling ☺️
